pkgs/development/python-modules/deezer-python/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, requests, tornado }:
```

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
, isPy3k
, requests
, tornado
}:
```

One per line is preferred, since it makes diffs easier to follow.